### ISSUE TITLE
DO_TIMING enabled by default in sparsify_fps

### ISSUE THREAD

**Issue by Luthaf** [cosmo-epfl/librascal#211]
It looks like timing code is enabled by default in sparsify_fps:

https://github.com/cosmo-epfl/librascal/blob/c9523a8f7b14e0a3cd588ed459e81ece4340487d/src/rascal/utils/sparsify_fps.cc#L30-L36

What is the rational for this? Was it development code that should be removed, or should we keep this around (maybe disabling it by default) ?

**Comment by Luthaf** [comment]
Ping @ceriottm, the `#ifdef` was introduced in dccc9b744f9fc4247ed1a5036780e55bb0d0f53e, and the code was here since b5b301a5a42870aa71d224a33e23a9dd3c367893.

**Comment by felixmusil** [comment]
it is the first attempt at profiling the code. It should be disabled by default and maybe converted to the google benchmark format when it is out.

**Comment by ceriottm** [comment]
Can be removed. Also, FPS should be moved out of librascal, so I wouldn't
spend too much time on it right now.

**Comment by Luthaf** [comment]
Should FPS be a part of the `rascal-models` library? (ref #199)

**Comment by ceriottm** [comment]
yes, or something like rascal-tools. this is a discussion we need to have,
sometimes...
